debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -3215,6 +3276,7 @@ public void testShouldNotCloseConnectionFetchingMetadataWithNewDataTypes() throw
         System.out.println(recordsForTopic.get(0));
     }
 
+    @Ignore("YB: updating replica identity unsupported")
```

Comment:
Could you mention the list of tests related to ALTER REPLICA IDENTITY that would need to be enabled once the support for  ALTER REPLICA IDENTITY is made available in the ticket for Update support

Reply:
Done.